<launch>
    <include file="$(find gl_ros_driver)/launch/gl_ros_driver.launch" />

    <node pkg="tresc3_scan_algorithm" type="tresc3_scan_node" name="tresc3_scan_node" output="screen">
        <param name="frame_id" type="string" value="pcl_data"/>
        <param name="sub_topicname_lidar" type="string" value="scan"/>
        <param name="pub_topicname_pcl" type="string" value="pcl_data"/>
        <param name="direction" type="string" value="$(arg direction)"/>
        <param name="velocity" type="double" value="$(arg velocity)"/>
        <param name="height" type="double" value="$(arg height)"/>
    </node>
</launch>
